src/transport/CryptoContext.cpp

```diff
@@ -131,16 +130,13 @@ CHIP_ERROR CryptoContext::InitFromKeyPair(const Crypto::P256Keypair & local_keyp
     return InitFromSecret(ByteSpan(secret, secret.Length()), salt, infoType, role);
 }
 
-CHIP_ERROR CryptoContext::GetIV(const PacketHeader & header, uint8_t * iv, size_t len)
+CHIP_ERROR CryptoContext::BuildIV(IVView iv, uint8_t securityFlags, uint32_t messageCounter, NodeId nodeId)
```

Comment:
IV is the wrong name in the API altogether. The spec calls it Nonce, and the AES-CCM mode of operation described in NIST SP 800-38A also uses "N" and "nonce". "IV" should have never been used as terminology. I suggest that if this code is touched, we make sure to refactor to use proper terminology.

Reply:
Address separately #16216

Will rename IV in this PR soon.